src/evict/evict_lru.c

```diff
@@ -315,7 +315,7 @@ __evict_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
     /*
      * Cache a history store cursor to avoid deadlock: if an eviction thread marks a file busy and
      * then opens a different file (in this case, the HS file), it can deadlock with a thread
-     * waiting for the first file to drain from the eviction queue. See WT-5946 for details.
```

Comment:
I didn't find WT-5946 to add important context to the description of the potential race already in the comment.